finalProject/fizzBuzz.js

```diff
@@ -0,0 +1,26 @@
+  var range = {
```

Comment:
Great work on arriving at a solution to this problem!

Interesting choice to use a class to set this up - you'll learn more about this in Mod 1, but classes can sometimes be a little heavy-handed for this particular use case. One thing I want you to review is how you can use parameters with a function to achieve the same thing you did with this class example! For context, functions are more lightweight than classes and in fact, classes are actually functions under the hood.

This could be a great opportunity to refactor this into a function that uses parameters instead of setting up an entire class.